src/components/Modal/Modal.js

```diff
@@ -202,6 +202,22 @@ export default class Modal extends Component {
     }
   }
 
+  componentDidMount = () => {
+    if (!this.props.open) {
+      return;
+    }
+    const primaryFocusElement = this.innerModal.current.querySelector(
```

Comment:
Are we guaranteed that the refs will have `current` on them during this pass or do we need to assert they exist before calling/using them?

Reply:
it seems we are not checking if the `current` attribute exists in other parts of the component, and I think it's because we can expect it to be present when we are calling refs?

Reply:
Makes sense @emyarod! Wasn't sure where we were using inline callback refs versus named callback refs (or `React.createRef`) 😅